chore: More details to commit message (#1022)
medic/cht-docs: content/en/contribute/code/workflow.md

### Commit message format

The commit format should follow this [conventional-changelog angular preset](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular). This means we can automate some of the release process. See the list of commit types and examples below:

Type | Description | Example commit message | Release type
-- | -- | -- | --

> Fixes | A bug fix. | fix(#123): infinite spinner when clicking contacts tab twice | patch

Reviewer: ```suggestion
Bug fixes | Change code that wasn't working as intended. | fix(#123): infinite spinner when clicking contacts tab twice | patch
```

Performance | A code change that improves performance. Measure the performance improvement to inform the community. | perf(#789): lazily loaded angular modules | patch

> Features | A new feature or change in the code that users will notice. | feat(#456): add home tab | minor

Reviewer: ```suggestion
Features | A new feature or improvement that users will notice. | feat(#456): add home tab | minor
```

> Non-code | A change that user won't notice, like a change in a README file, adding e2e tests, update dependencies, removing unused code, etc. | chore(#123): update README | none

Reviewer: Update for consistent tense.

```suggestion
Non-code | A change that user won't notice, like a change in a README file, adding e2e tests, updating dependencies, removing unused code, etc. | chore(#123): update README | none
```

The commit format should follow this [conventional-changelog angular preset](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular). This means we can automate some of the release process. Examples are provided below.

Type | Description | Example commit message | Release type
-- | -- | -- | --
Bug fixes | A bug fix. | fix(#123): infinite spinner when clicking contacts tab twice | patch
Performance | A code change that improves performance. Measure the performance improvement to inform the community. | perf(#789): lazily loaded angular modules | patch
Features | A new feature or change in the code that users will notice. | feat(#456): add home tab | minor
Non-code | A change that user won't notice, like a change in a README file, adding e2e tests, update dependencies, removing unused code, etc. | chore(#123): update README | none

> Breaking | Breaking changes should be explained under performance type. | perf(#2): remove reporting rates feature <br/> BREAKING CHANGE: reporting rates no longer supported | major

Reviewer: "Breaking" is special because it could apply to any of the above types. "perf" was just this example, but it's highly likely to be a "feat" but could even be a "fix". Maybe we need to split it out of the table to make this clear?

Author: I've created a note and added that piece there. What do you think?
